index.html

```diff
@@ -150,7 +150,31 @@ <h6>languaages</h6>
                 </li>
                  </ul>
             </section>
+            <section class="contact">
+              <h1 class="me">Contact me</h1>
+              <p>If you have an application you
+                are interested in developing, a
+                feature that you need built or aproject that needs coding. 
+                
+                I’d love to help with it!</p>
+                <form class="contact_form">
+                     <label for=""></label>
+                    <input type="text" placeholder="Lorem Ipsum" value="" class="form1">
+
+                    <input type="email" placeholder="loremipsum@gmail.com" class="form2">
+
+                    <input type="text" placeholder="write your massage here" class="form3">
+                    
```

Comment:
- Nice work so far 👍🏿
- [ ]  I notice you are using an `input`  please i kindly suggest you use a textarea. using a` <textarea>` instead of an `<input>` element can be more appropriate in certain scenarios, especially when expecting multiline input or larger blocks of text.
Considering your suggestion, I recommend replacing the `<input>` element with a `<textarea>` element to allow for easier input of multiple lines or larger amounts of text.
- [ ]  I can see that the "required" attribute is indeed not present in the provided code. The inclusion of the "required" attribute ensures that the user must fill out the input field before submitting the form, meeting the requirement for a required field.
- [ ] Missing maxlength attribute: The code does not include the "maxlength" attribute, which may be necessary to enforce a specific character limit on the input field. It is important to clarify if there is a maximum length requirement and add the appropriate "maxlength" value.
- [ ] I can see that the "required" attribute is indeed not present in the provided code. The inclusion of the "required" attribute ensures that the user must fill out the input field before submitting the form, meeting the requirement for a required field.